.circleci/config.yml

```diff
@@ -1,17 +1,26 @@
-version: 2
+version: 2.1
 
+# -------------------------
+#   PARAMETERS
+# -------------------------
+parameters:
+  cache_key_version:
+    type: string
+    default: "v1"
+
+  circleci_service_account:
+    type: string
+    default: "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com"
```

Comment:
It can be hidden with either a [context](https://app.circleci.com/settings/organization/github/all-of-us/contexts?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fall-of-us%2Fworkbench%2F510%2Fworkflows%2Ff3fe3af4-2cb4-44bf-8640-803fd7765921%2Fjobs%2F79899%2Fsteps) or a [env variable](https://app.circleci.com/settings/project/github/all-of-us/workbench/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fall-of-us%2Fworkbench)
I can do change here or in another PR. Do you have any preference on how and when?